### ISSUE TITLE
ESP needs a reboot after first OTA flash

### ISSUE THREAD

**Issue by jbaldwinroberts** [resin-io-projects/esp8266#5]

**Comment by jbaldwinroberts** [comment]
This is a known issue with the ESP8266 - the workaround is to ensure the ESP8266 is reset / power cycled after the initial serial flash.
https://github.com/esp8266/Arduino/issues/2876
https://github.com/esp8266/Arduino/issues/1722
https://github.com/esp8266/Arduino/issues/2478

@brownjohnf @mccollam please can you try the work around above and see if the problem persists.

**Comment by brownjohnf** [comment]
Yeah, I'll try it some time. Not sure when I'll have a minute though.
